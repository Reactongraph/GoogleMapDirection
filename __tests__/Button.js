/**
 * @format
 * @lint-ignore-every XPLATJSCOPYRIGHT1
 */

import 'react-native';
import React from 'react';
import Button from '../src/components/Button';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('button snapshots', () => {
  const snap = renderer.create(
    <Button />
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
