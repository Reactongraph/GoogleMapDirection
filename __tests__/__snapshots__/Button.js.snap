// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button snapshots 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <Text />
</View>
`;
